content/en/blog/community/spinkube-kind-rd/index.md

```diff
@@ -0,0 +1,328 @@
+  - hostPath: /root/kind-spin/config.toml
+    containerPath: /etc/containerd/config.toml
+  - hostPath: /usr/local/containerd-shims/containerd-shim-spin-v2
+    containerPath: /usr/local/bin/containerd-shim-spin-v2
+"@ | rdctl shell -- tee -a /root/kind-spin.yaml
+```
+
+**Linux and MacOS Users ONLY**
+
+```yaml
+cat <<EOF > $HOME/kind-spin.yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+  - hostPath: ${HOME}/config.toml
+    containerPath: /etc/containerd/config.toml
+  - hostPath: /usr/local/containerd-shims/containerd-shim-spin-v2
+    containerPath: /usr/local/bin/containerd-shim-spin-v2
+EOF
+```
+
+#### Connecting the dots (two)
+
+Rancher Desktop leverages two different technologies depending on the OS its installed.
+
+On Windows, [WSL2](https://learn.microsoft.com/en-us/windows/wsl/install) will be used and for Linux and MacOS, [Lima](https://lima-vm.io/docs/) is the preferred choice.
+
+While both technologies run Linux in a microVM, the behaviors differ in some parts. And the mountpoints with the host system are one of these differences.
+
+In the case of RD on WSL, the file generated is created **inside** the microVM, as nerdctl will need to have acceess to the file's path. Technically speaking, the mountpoint `/mnt/c` could also be used, however sometimes it's not available due to WSL main configuration. This way should be a bit more generic.
+
+Concerning RD on Lima, `$HOME` is mounted inside the microVM, therefore nerdctl will already see the files, and there's not need on copying the files over like it's done for WSL.
+
+Finally, on both cases, the binary `containerd-shim-spin-v2` is already accessible inside the microVMs.
+
+## Create KinD cluster
+
+With all the preparations done, you can now create a new KinD cluster as follows:
+
+**Windows Users ONLY**
+
+```powershell
+# Set the KinD provider to nerdctl
+$env:KIND_EXPERIMENTAL_PROVIDER="nerdctl"
+
+# Create a new cluster with the config file "kind-spin.yaml"
+kind create cluster --config=$HOME/kind-spin.yaml
+```
+
+![KinD create cluster on Windows](wsl-kind-create-cluster.png)
+
+**Linux and MacOS Users ONLY**
+
+```bash
+# Set the KinD provider to nerdctl
+export KIND_EXPERIMENTAL_PROVIDER=nerdctl
+
+# Create a new cluster with the config file "kind-spin.yaml"
+kind create cluster --config=$HOME/kind-spin.yaml
+```
+
+![KinD create cluster on *nix](lima-kind-create-cluster.png)
+
+Now that you have a KinD cluster running with the spin plugin enabled for `containerD`. However, it is not yet used by Kubernetes (`runtimeClass`). This will be done on the next section.
+
+## Deploy SpinKube
+
+From here, you can reference the [excellent quickstart to deploy SpinKube](https://www.spinkube.dev/docs/spin-operator/quickstart/) for a detailed explanation of each step.
+
+To avoid repetition, and to encourage you to go read the quickstart (and the overall SpinKube docs), the steps below will only include short descriptions.
```

Comment:
```suggestion
To avoid repetition, and to encourage you to go read the quickstart (and the overall SpinKube docs), the steps below will only include short descriptions:
```